nvflare/private/fed/server/run_manager.py

```diff
@@ -33,10 +35,10 @@ def __init__(self, job_id, app_path):
         self.job_id = job_id
         self.start_time = time.time()
         self.app_path = app_path
-        self.status = MachineStatus.STOPPED
+        self.status = MachineStatus.STOPPED.value
 
 
-class RunManager:
+class RunManager(EngineSpec):
```

Comment:
Just a side note I think we need to clean up these names make server and client side names more consistent.

For client side we have ClientEngine inherit `ClientEngineInternalSpec`
ClientRunManager inherit `ClientEngineExecutorSpec`.

On server side we have ServerEngine inherit `ServerEngineInternalSpec`
ServerRunManager inherit `EngineSpec`


And then the `ServerEngineInternalSpec` inherits `ServerEngineSpec` inherits `EngineSpec`.
`ClientEngineInternalSpec` inherits `ClientEngineSpec` inherits `EngineSpec`.
`ClientEngineExecutorSpec` inherits both `ClientEngineSpec` and `EngineSpec`

^ we need some clean up for these complicated relationships/naming.
Maybe some middle layers can be removed.

Reply:
EngineSpec defines the operations which needed for both server and client sides, like validate_targets(). Server / Client have its own spec to define the operations which needed for parent and child process. The interface hierarchy is needed. We may name the interface a little differently, but keep the naming for now.